.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: erlang
```

Comment:
I believe you have to allow Travis CI to "read" this repo. in order for tests to run.